lib/api.js

```diff
@@ -48,6 +49,40 @@ function removeCoupling(heroku, app) {
            .then(coupling => deleteCoupling(heroku, coupling.id));
 }
 
+function getPipelineCouplings(heroku, pipelineId) {
+  return heroku.request({
+    method: 'GET',
+    path: `/pipelines/${pipelineId}/pipeline-couplings`,
+    headers: { 'Accept': V3_HEADER }
+  });
+}
+
+function getAppFitler(heroku, appIds) {
+  return heroku.request({
+    method: 'POST',
+    path: `/filters/apps`,
+    headers: { 'Accept': FILTERS_HEADER },
+    body: { in: { id: appIds } }
+  });
+}
+
+function getPipelineApps(heroku, pipelineId) {
```

Comment:
I actually think we keep it here - `getApps` or `listApps` feels like it will get a full app list rather than a list scoped to my pipeline. I will change it to `listPipelineApps` though